doc/mgr/rgw.rst

```diff
@@ -18,36 +19,111 @@ RGW Realm Operations
 
 Bootstrapping RGW realm creates a new RGW realm entity, a new zonegroup,
 and a new zone. It configures a new system user that can be used for
-multisite sync operations, and returns a corresponding token. It sets
-up new RGW instances via the orchestrator.
+multisite sync operations. Under the hood this module instructs the
+orchestrator to create and deploy the corresponding RGW daemons. The module
```

Comment:
@adk3798 I tested this case and right now the module create the realm/zonegroup/zone and then fails to create the daemons showing the following error:

```
[ceph: root@ceph-node-0 /]# ceph rgw realm bootstrap -i test.yaml 

Error EINVAL: Traceback (most recent call last):
  File "/usr/share/ceph/mgr/orchestrator/_interface.py", line 1567, in _oremote
    return mgr.remote(o, meth, *args, **kwargs)
  File "/usr/share/ceph/mgr/mgr_module.py", line 2147, in remote
    args, kwargs)
ImportError: Module not found

```

I think it would be better to check if the orchestrator is available or not before proceeding with the realms creation. WDYT?

Reply:
@adk3798 I added a check to make sure cephadm is available before running the cmds.